authentication-guides/servicenow/README.md

```diff
@@ -0,0 +1,152 @@
+---
+name: ServiceNow   # The name of the guide
+description: Connect Creator Studio to ServiceNow   # Brief description of the guide
+time_in_minutes: 30   # Time in minutes to complete the guide
+difficulty_level: Intermediate  # Difficulty level: "Beginner", "Intermediate", or "Advanced"
+---
+
+# **Introduction**
+
+ServiceNow provides granular table-level access to its data as part of its service management capabilities. This guide will walk through how you can connect your ServiceNow instance to Moveworks Creator Studio with basic auth and OAuth 2 credentials, and test it in Creator Studio.
+
+# **Prerequisites**
+
+- Access to a ServiceNow Instance (Sandbox or Production)
+- [Install Postman](https://www.postman.com/downloads/)
+
+# **Connect with Basic Authentication**
+
+## **Step 1: Get your developer username and password**
+
+1. If you’re using a personal developer instance
+    1. Go to the [ServiceNow Developer Portal](https://developer.servicenow.com/dev.do)
+    2. Click on your avatar > `Manage Instance Password`
+        
+        ![Untitled](Authentication%20Tutorial%20ServiceNow%20v2%20bf9283817e704a4385a4c5e8ffe8859e/Untitled.png)
+        
+    3. Save your `Instance name`, `Username`, and `Password`
+        
+        ![Screenshot 2024-02-29 at 8.58.42 AM.png](Authentication%20Tutorial%20ServiceNow%20v2%20bf9283817e704a4385a4c5e8ffe8859e/Screenshot_2024-02-29_at_8.58.42_AM.png)
+        
+2. If you’re using a production or implementation tenant owned by your organization
+    1. Create a service account user (`username` & `password`), note it down
+    2. Your instance name is the prefix of your URL: `https://{{instance_name}}.service-now.com`
+
+## Step 2: Grant the appropriate permissions for your service account
+
+Follow steps in our help site to grant the necessary roles/permissions required to execute the API calls:
+
+[https://help.moveworks.com/docs/servicenow-access-requirements#service-account-requirements--permissions](https://help.moveworks.com/docs/servicenow-access-requirements#service-account-requirements--permissions)
+
```

Comment:
Swap for https://docs.servicenow.com/bundle/tokyo-platform-administration/page/administer/roles/reference/r_BaseSystemRoles.html